lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -907,7 +907,7 @@ def _from_file(self, root_path: Path, file_path: Path) -> List[dict]:
         with file_path.open() as rf:
             # Load a list of rules from file then add labels and filters
             try:
-                rule_file = yaml.safe_load(rf)
+                rule_file = yaml.safe_load(rf) or {}
```

Comment:
Sounds to me like a case in why we would log at least a debug. It is unlikely to be intended to have an empty rule file.

Reply:
An ERROR is logged in this case, line 924.
This behavior (logging an error) is also anchored in the utests.

Reply:
L924 says that it is invalid. It does not say _why_.

Reply:
Added.